package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Sushain Cherivirala <sushain@skc.name>",
   "license": "GPL-3.0-or-later",
```

Comment:
Can you update the `version` field above to actually match the latest released GH version if we're going to use it as the canonical source?